openai/dto/OpenAiCompletionResponseDTO.ts

```diff
@@ -1,14 +1,14 @@
 // Copyright (c) 2023. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
 
 import {
-    explainOpenAiCompletionResponseChoice, explainOpenAiCompletionResponseChoiceOrError,
-    isOpenAiCompletionResponseChoice, isOpenAiCompletionResponseChoiceOrError,
+    explainOpenAiCompletionResponseChoiceOrError,
+    isOpenAiCompletionResponseChoiceOrError,
     OpenAiCompletionResponseChoice
 } from "./OpenAiCompletionResponseChoice";
 import { OpenAiModel } from "../types/OpenAiModel";
 import { explainRegularObject, isRegularObject } from "../../types/RegularObject";
 import { explainNoOtherKeys, hasNoOtherKeys } from "../../types/OtherKeys";
-import { explainString, isString } from "../../types/String";
+import {explainString, explainStringOrUndefined, isString, isStringOrUndefined} from "../../types/String";
```

Comment:
Style nit: whitespace disappears after and before the braces, which all the other imports still have.

Reply:
Fixed 👍